src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioIntegrationTestUtilities.csproj

```diff
@@ -104,6 +105,10 @@
       <Project>{e2e889a5-2489-4546-9194-47c63e49eaeb}</Project>
       <Name>Diagnostics</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\Test\Utilities\Portable\TestUtilities.csproj">
```

Comment:
Yes, we want to make this a standalone library (moved to its own repository) so that multiple repositories can easily consume.

There are probably multiple things in TestUtilities that are useful for sharing across our repositories as well, so maybe we need a general TestUtilities repository...